jmclient/jmclient/blockchaininterface.py

```diff
@@ -462,12 +462,14 @@ def estimate_fee_per_kb(self, N):
         if not rpc_result:
             # in case of connection error:
             return None
-        mempoolminfee_in_sat = btc.btc_to_sat(rpc_result['mempoolminfee'])
-        mempoolminfee_in_sat_randomized = random.uniform(
-            mempoolminfee_in_sat, mempoolminfee_in_sat * float(1.2))
 
         tx_fees_factor = float(jm_single().config.get('POLICY',
             'tx_fees_factor'))
```

Comment:
IMO we should handle negative values here, as it possible for user to configure that way. Without this change `mempoolminfee_in_sat_randomized` could end up being lower than `mempoolminfee_in_sat`.

```suggestion
        tx_fees_factor = abs(jm_single().config.getfloat('POLICY', 'tx_fees_factor'))
```